nova/scheduler/filters/external_scheduler_filter.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2024 SAP SE or an SAP affiliate company and CobaltCore contributors.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+
+import requests
+from oslo_log import log as logging
+import nova.conf
+from nova.filters import BaseFilter
+
+
+LOG = logging.getLogger(__name__)
+CONF = nova.conf.CONF
+
+
+class ExternalSchedulerFilter(BaseFilter):
+    """Filter using an out of tree scheduler via a REST API."""
+
+    RUN_ON_REBUILD = False
+
+    def __init__(self):
+        self.api_url = CONF.filter_scheduler.external_scheduler_api_url
```

Comment:
Why don't we call `super().__init__(self)` here?

Reply:
true. done